frontend/test/snapshot-creators/qa-db.cy.snap.js

```diff
@@ -28,6 +29,20 @@ describe("qa databases snapshots", { tags: "@external" }, () => {
     snapshot("mongo-4");
     deleteDatabase("mongoID");
 
+    restoreAndAuthenticate();
+
+    setupTestDB("mysql");
+    addMySQLDatabase("Writable MySQL8", true);
+    snapshot("mysql-writable");
+    deleteDatabase("mysqlID");
+
+    restoreAndAuthenticate();
+
+    setupTestDB("postgres");
+    addPostgresDatabase("Writable Postgres12", true);
+    snapshot("postgres-writable");
+    deleteDatabase("postgresID");
+
```

Comment:
Do you maybe have an info handy re: how much time does this add to the snapshot generation?

Reply:
FWIW, my first attempt to run Cypress on this branch failed on the snapshot generation step with "the sync too long".

It doesn't have to mean anything, but we should keep an eye on that.

Reply:
> Do you maybe have an info handy re: how much time does this add to the snapshot generation?

I just tested and it added 2 seconds on my machine: 13s -> 15s.